### DISCUSSION TITLE
局域网

### DISCUSSION THREAD

**Discussion by xtayssqumpd** [AlistGo/alist#3487] [Q&A]
请问下，用Mac或者安卓手机搭建alist，在infuse只能在局域网下观看吗

**Comment by chenjuefei** [comment]
你有公网IP么。有的话路由器转发，外网能访问到。网速够。就可以吧

**Reply by xtayssqumpd** [reply]
没有公网IP，记得前段时间外网还看了，最近只能在局域网看了
